lib/secure-hash.js

```diff
@@ -53,7 +53,7 @@ module.exports = function (base64, crypto, promise) {
      * @param {secureHash~config} [config]
      * @return {Promise.<string>}
      */
-    function secureHashAsync(data, config) {
+    function secureHashAsync(data, config, regularEncode) {
```

Comment:
Made this into two different functions to encode into base64 and one to encode into base64 but for a URI so it doesn't have slashes and pluses.